compiler/dyno/lib/resolution/scope-queries.cpp

```diff
@@ -249,39 +249,57 @@ const Scope* scopeForId(Context* context, ID id) {
 
 static bool doLookupInScope(Context* context,
                             const Scope* scope,
+                            const VisibilityPath* path,
                             UniqueString name,
                             LookupConfig config,
                             std::unordered_set<const Scope*>& checkedScopes,
                             std::vector<BorrowedIdsWithName>& result);
 
 static bool doLookupExprInScope(Context* context,
                                 const Scope* scope,
+                                const VisibilityPath* path,
                                 const AstNode* expr,
                                 LookupConfig config,
                                 std::unordered_set<const Scope*>& checkedScopes,
                                 std::vector<BorrowedIdsWithName>& result,
                                 UniqueString& name,
                                 const Scope*& resultScope);
 
-static const ResolvedVisibilityScope*
-  partiallyResolvedVisibilityScope(Context* context, const Scope* scope);
+static const std::vector<VisibilitySymbols>&
+resolveVisibilityStmt(Context* context, ID useImportId,
+                      VisibilityPath path);
 
 static bool doLookupInImports(Context* context,
                               const Scope* scope,
+                              const VisibilityPath& path,
```

Comment:
Why not type this as `const VisibilityPath*` to be consistent with the other two above?

Reply:
I didn't want this function to have to worry about the possibility that it is nullptr